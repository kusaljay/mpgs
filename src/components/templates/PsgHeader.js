import React from 'react';
import { Link } from 'react-router-dom';
import fbase from '../fbase';

const PsgHeader = () => {
  return (
    <header className="header-wrapper">
      <div className="container d-flex justify-content-between align-items-center inner-wrapper">
        <img src="" className="" alt="Logo" />
        <Link className="psg-link-text" onClick={() => fbase.auth().signOut()}>Logout</Link>
      </div>
    </header>
  )
}

export default PsgHeader;
