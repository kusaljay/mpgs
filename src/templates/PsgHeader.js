import React from 'react';
import { Link } from 'react-router-dom';
import firebase from '../utils/firebase';
/* import logo from '../assets/logo-gradient.svg'; */

const PsgHeader = () => {
  return (
    <header className="header-wrapper">
      <div className="container d-flex justify-content-end align-items-center inner-wrapper">
        {/* <img src={logo} className="logo inner" alt="myob logo" /> */}
        <Link className="psg-link-text" to="./login" onClick={() => firebase.auth().signOut()}>Logout</Link>
      </div>
    </header>
  )
}

export default PsgHeader;
