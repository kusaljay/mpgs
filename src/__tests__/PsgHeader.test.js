import React from 'react';
import { shallow } from 'enzyme';
import PsgHeader from '../components/templates/PsgHeader';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<PsgHeader />);
});

describe ('PsgHeader', () => {
  it ('Should render header', () => {
    expect(wrapper).toMatchSnapshot();
  });

});

