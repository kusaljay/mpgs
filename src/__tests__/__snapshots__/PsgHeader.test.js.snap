// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PsgHeader Should render header 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PsgHeader />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <div
        className="container d-flex justify-content-between align-items-center inner-wrapper"
      >
        <img
          alt="myob logo"
          className="logo inner"
          src="logo-gradient.svg"
        />
        <Link
          className="psg-link-text"
          onClick={[Function]}
          to="../components/login"
        >
          Logout
        </Link>
      </div>,
      "className": "header-wrapper",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <img
            alt="myob logo"
            className="logo inner"
            src="logo-gradient.svg"
          />,
          <Link
            className="psg-link-text"
            onClick={[Function]}
            to="../components/login"
          >
            Logout
          </Link>,
        ],
        "className": "container d-flex justify-content-between align-items-center inner-wrapper",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "myob logo",
            "className": "logo inner",
            "src": "logo-gradient.svg",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "Logout",
            "className": "psg-link-text",
            "onClick": [Function],
            "to": "../components/login",
          },
          "ref": null,
          "rendered": "Logout",
          "type": [Function],
        },
      ],
      "type": "div",
    },
    "type": "header",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="container d-flex justify-content-between align-items-center inner-wrapper"
        >
          <img
            alt="myob logo"
            className="logo inner"
            src="logo-gradient.svg"
          />
          <Link
            className="psg-link-text"
            onClick={[Function]}
            to="../components/login"
          >
            Logout
          </Link>
        </div>,
        "className": "header-wrapper",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <img
              alt="myob logo"
              className="logo inner"
              src="logo-gradient.svg"
            />,
            <Link
              className="psg-link-text"
              onClick={[Function]}
              to="../components/login"
            >
              Logout
            </Link>,
          ],
          "className": "container d-flex justify-content-between align-items-center inner-wrapper",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "alt": "myob logo",
              "className": "logo inner",
              "src": "logo-gradient.svg",
            },
            "ref": null,
            "rendered": null,
            "type": "img",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {
              "children": "Logout",
              "className": "psg-link-text",
              "onClick": [Function],
              "to": "../components/login",
            },
            "ref": null,
            "rendered": "Logout",
            "type": [Function],
          },
        ],
        "type": "div",
      },
      "type": "header",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
